src/app/tabs/resolve/components/ResolveComponent.js

```diff
@@ -14,7 +14,7 @@ import UserWaitingComponent from '../../../components/UserWaitingComponent';
 const renderResolutions = (supportedInterfaces) => {
   const hasAddr = supportedInterfaces.indexOf('addr') > -1;
   const hasChainAddr = supportedInterfaces.indexOf('chainAddr') > -1;
-  const hasMulticoin = supportedInterfaces.indexOf('multicoin') > -1;
+  const hasMulticoin = false;
```

Comment:
Is it possible to not change this code, and remove 'multicoin' from 'supportedInterfaces' array?
That way it would still work in the future if multicoin is added

Reply:
I thought of it but I think removing it from [here](https://github.com/rsksmart/rns-manager-react/blob/b26a4d9cacf51df022aaadcfb1de06c354f75cb9/src/app/tabs/newAdmin/resolver/operations.js#L166) would be harder to track back in the future

Reply:
Hey guys,
1. I didn't though on change it, to keep the changes only related to the issue on this PR, but I can added.
2. Since you both agreed on removing, and not leave it as temporal, I'll removing it from `supportedInterfaces.json`.

Reply:
On this, because we won't actively keep maintaining this version of the project, I suggest we have a new PR that'll address this change the way @antomor and @DinizSa have suggested to the develop branch.